### ISSUE TITLE
Sending an image taken from Twidere (in an answer to self) results in "Sorry, page does not exist".

### ISSUE THREAD

**Issue by orithena** [TwidereProject/Twidere-Android#898]
If I send a tweet with an image that has been taken via Twideres "add attachment > take photo > use Camera App to take photo" (sorry, re-translated from german), the tweet is saved to drafts while a toast pops up: "Sorry, that page does not exist" -- at least when replying to self.

## Expected behavior
The tweet (answer to self) gets posted including the image.

## Actual behavior
Tweet does not get posted, instead it gets saved to drafts (where I don't even see an option to save that photo I just took) and a toast pops up saying "Sorry, that page does not exist".

## Steps to reproduce
0. (Possibly) answer to own tweet.
1. Hit Clippy.
2. Choose "Foto aufnehmen" ("Take photo"?) from popup menu.
3. Choose Androids default Camera App (if you're being asked... I am).
4. Take photo, confirm.
5. Add some text to tweet, send.

## Extra info

Android version: 6.0.1

Network type:

- [ ] Cellular
- [x] Wi-Fi

App version: 3.6.21

Build variant:

- [x] Google
- [ ] F-Droid

Micro-blogging service:

- [x] Twitter
- [ ] Fanfou
- [ ] GNU Social / StatusNet
- [ ] Mastodon

App settings:

- [ ] Custom API
- [ ] Custom consumer key/secrets
- [x] Official keys
- [ ] Using [Thumbor](https://github.com/thumbor/thumbor)
- [ ] Connected through proxy
- [ ] Media preview enabled

**Comment by mariotaku** [comment]
I can't reproduce. Are you replying to a retweet or something?

**Comment by orithena** [comment]
I can't reproduce either anymore. Even the tweets that got saved to drafts are now being sent correctly (which also didn't work at the time of raising this issue).

I'm now writing it off to a temporary problem on twitters side or on my phone (I did reboot since then). It's just that I used that function for the first time (normally, I'd attach images from my gallery).

Thanks for investigating, though!
